sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blobs/tests/Azure.Messaging.EventHubs.CheckpointStore.Blobs.Tests.csproj

```diff
@@ -7,6 +7,10 @@
     <ExcludeRecordingFramework>true</ExcludeRecordingFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="..\..\Azure.Messaging.EventHubs\tests\Shared\*.cs" />
```

Comment:
Would you please be so kind as to rename the folder to `SharedSource` to match the convention used with other cross-project sharing within the Event Hubs ecosystem?

Reply:
What do you think the "Source" adds to the name? It seems kind of redundant to me. I will think about the name a little more and try to come up with one that makes since given the fact I'm going to move it to the common folder.

Reply:
Do you think that we could do that in a different context, and preferably after GA for Event Hubs?   I'm blocked on release at present and looking at a schedule for GA delivery with very, very little room.

Reply:
I like "source" in this case to denote that this is shared as a source file and to imply the semantics of "you change this here, and other things change too."    I'm happy to discuss naming at some point, but I'd like to push this until after we GA in December due to tight timelines.

For now, since we'll be local to Event Hubs, I'd like to maintain control of the naming as the library owner.  I'm happy to discuss changes to it at the point we move it outside of the Event Hubs scope.

Reply:
Just a note that I'm fine with "Shared" being the folder in the tests library that is doing the sharing.  It's the destination that is borrowing things that I'd like to ensure that semantics are clear for.

Reply:
... what I'm asking for is this, if you would:

```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <AssemblyName>Azure.Messaging.EventHubs.CheckpointStore.Blobs.Tests</AssemblyName>
    <VersionPrefix>1.0.0</VersionPrefix>
    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
    <ExcludeRecordingFramework>true</ExcludeRecordingFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.NET.Test.Sdk" />
    <PackageReference Include="System.Net.WebSockets.Client" />
    <PackageReference Include="System.ValueTuple" />
    <PackageReference Include="NUnit" />
    <PackageReference Include="NUnit3TestAdapter" />
    <PackageReference Include="Moq" />
    <PackageReference Include="Polly" />
    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" />
    <PackageReference Include="Microsoft.Azure.Management.EventHub" />
    <PackageReference Include="System.Threading.Tasks.Extensions" />
    <PackageReference Include="Microsoft.Azure.Management.Storage" />
    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" />
    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" />
  </ItemGroup>

  <ItemGroup>
    <ProjectReference Include="..\src\Azure.Messaging.EventHubs.CheckpointStore.Blobs.csproj" />
    <ProjectReference Include="..\samples\Azure.Messaging.EventHubs.CheckpointStore.Blobs.Samples.csproj" />
  </ItemGroup>

  <!-- Import the shared infrastructure from the Event Hubs client tests (shared source) -->
  <ItemGroup>
    <Compile Include="..\..\Azure.Messaging.EventHubs\tests\Shared\*.cs" Link="SharedSource\%(Filename)%(Extension)" />
  </ItemGroup>
</Project>
```

Reply:
Thanks for the clarification I didn't realize you were meaning the folder view in VS. I updated to use your sample.